incompatibility/incomp-scanning_test.go

```diff
@@ -62,7 +65,7 @@ func TestBasicSecurityIncompatibility(t *testing.T) {
 	}
 }
 
-//Assert that all reported incompatibility paths can be found when travering file
+// Assert that all reported incompatibility paths can be found when travering file
```

Comment:
Github won't let me comment on ln 50: Should this test be in this file?  Would this not be more appropriate in the reporter?

Reply:
removed as this was a early version of reporter testing and is covered in report_testing.go